taxonomies/categories.txt

```diff
@@ -45566,6 +45645,14 @@ ciqual_food_code:en:13025
 ciqual_food_name:en:Mango, pulp, raw
 ciqual_food_name:fr:Mangue, pulpe, crue
 
+<en:Raw mango pulp
```

Comment:
It should be under "en:mangoes", not mango pulp.

Reply:
I don't understand what their "pulp, raw" means. Are they talking about the fruit ? The fruit as pulp ?

Reply:
"Mongo, raw, pulp" = the pulp of the mango fruit (i.e. without the kernel).

But if you create a category "Mangue importée par avion", it should be under "Mangues". Or rename the category "Pulpe de mangue importée par avion".